arwen/elrondapi/bigIntOps.go

```diff
@@ -1074,6 +1233,10 @@ func v1_4_bigIntFinishSigned(context unsafe.Pointer, referenceHandle int32) {
 	output := arwen.GetOutputContext(context)
 	metering := arwen.GetMeteringContext(context)
 	runtime := arwen.GetRuntimeContext(context)
+	initialGasLeft := metering.GasLeft()
+	defer func() {
+		runtime.TraceGasUsed("bigIntFinishSigned", initialGasLeft)
```

Comment:
define all these string as constants please

Reply:
there would be over 100 string constants if I do that. I know it's because of the linter's warnings probably (mine doesn't show such warnings), but do we really want 100+ string constants?

Reply:
yes. in terms of compilation and execution speed it will be much faster.